mergify_engine/actions/queue.py

```diff
@@ -140,16 +143,31 @@ async def run(
             else:
                 status = await merge_base.get_rule_checks_status(
                     ctxt,
+                    ctxt.pull_request,
                     queue_rule_evaluated,
                     unmatched_conditions_return_failure=False,
                 )
             await car.update_summaries(
                 status, status, queue_rule_evaluated, will_be_reset=need_reset
             )
+        elif car and car.state == "created":
+            if not ctxt.has_been_only_refreshed():
+                # NOTE(sileht): It's not a only refreshed, so we need to
+                # updated the associated transient pull request.
+                # This is mandatory to filter out refresh to avoid unless loop
+                # of refresh between this PR and the transient one
```

Comment:
```suggestion
                # NOTE(sileht): It's not only refreshed, so we need to
                # update the associated transient pull request.
                # This is mandatory to filter out a refresh to avoid useless loop
                # of refreshes between this PR and the transient one.
```